serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -0,0 +1,29 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-channels"]
+= Using Channels
+include::modules/common-attributes.adoc[]
+:context: serverless-channels
+
+toc::[]
+
+It is possible to sink events from an event source to a Knative Eventing channel.
```

Comment:
Admittedly, I know nothing about event sinks...is this the correct usage here? Don't know if "sink" in Serverless can also be a verb, or if it should be "sync" in this case?

Reply:
Yeah it's also a verb with a specific meaning :) should be OK here. Things like SinkBinding are explained in other parts of the docs for GA with our Eventing tech preview and the user should know what this means I think.